media-sound/audacity/audacity-3.2.2.ebuild

```
@@ -0,0 +1,241 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+WX_GTK_VER="3.2-gtk3"
```

(We like to have a blank space after EAPI)
```suggestion
EAPI=7

WX_GTK_VER="3.2-gtk3"
```